src/app/app.server.ts

```diff
@@ -0,0 +1,15 @@
+import express, { Express, Router} from 'express'
+import { router as breadRouter } from "./bread/bread.router"
```

Comment:
Bread entity is not part of the challenge, you can include other routers like `./cake/cake.router.ts`.